.changeset/tall-eyes-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tiptap/extension-drag-handle": major
+---
+
+Dismissable popup on keydown
```

Comment:
Could this be a bit more descriptive? I think it would be helpful to know which popup this is about and which keys will dismiss the popup

Reply:
Updated the changeset to more descriptive "Dismissable popup on keydown, when the popup is visible and the editor has focus, typing should hide the popup." does that work for you?